app/javascript/src/components/Dashboard/Notes/NotesMenuBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { Settings, Plus, Search } from "@bigbinary/neeto-icons";
+import { MenuBar } from "@bigbinary/neetoui/layouts";
+import { Typography } from "neetoui";
+
+const NotesMenuBar = ({ showMenu }) => (
```

Comment:
Why do we have to prefix with the `Notes` keyword for this component? What advantage does it provide?

Please read: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing